### DISCUSSION TITLE
Plot_posterior_predictive in HSSM

### DISCUSSION THREAD

**Discussion by oajilore** [lnccbrown/HSSM#326] [Q&A] [answered]
Hello,

Is there an HSSM equivalent to plot_posterior_predictive in HDDM?

Thanks,
Olu

**Comment by digicosmos86** [comment, marked as answer]
Hi @oajilore,

`plot_poterior_predictive` is now unofficially supported in the dev version of HSSM. If you install the dev version through git:

```sh
pip install git+https://github.com/lnccbrown/HSSM.git
```

You will have `plot_posterior_predictive` through `hssm.plotting.plot_posterior_predictive`. [This notebook](https://github.com/lnccbrown/HSSM/blob/main/docs/tutorials/plotting.ipynb) shows how to use it. We'll have more documentation for it once the next version is released.

**Reply by oajilore** [reply]
Great, thank you!
